redpanda/validators/validators.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+// Package validators contains generally useful validation functions for the Redpanda provider.
```

Comment:
[nit] I doubt you'll write something that's not useful haha
```suggestion
// Package validators contain validation functions for the Redpanda provider.
```

Reply:
Well, I specified generally useful because I wanted to exclude stuff like the ACLs validators which are useful in their specific case but not more generally. 

But... uh... reading it back it is pretty funny LOL